packages/web/src/components/stripe-on-ramp-modal/StripeOnRampModal.module.css

```diff
@@ -1,6 +1,6 @@
 .stripeModalWrapper {
   z-index: 99999999999999999999;
```

Comment:
I know this was an existing thing but...really? 😆 
Might as well make it `MAX_SAFE_INTEGER`...

Reply:
in css?

Reply:
Oh duh.
I was mostly just pointing out that this zIndex is unnecessarily large. Carry on.